src/styles/blocks/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__img {
+  align-self: center;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+}
+
+.card__code {
+  margin-top: 4px;
+
+  font-size: var(--text-size-secondary);
+  font-weight: var(--text-weight-secondary);
+  line-height: var(--text-line-height-secondary);
+  color: var(--text-color-secondary);
+}
+
+.card__rating {
+  display: flex;
+
+  margin: 16px 0 0;
+  justify-content: space-between;
+}
+
+.card__reviews {
+  font-size: var(--text-size-secondary);
+  font-weight: var(--text-weight-secondary);
+  line-height: var(--text-line-height-secondary);
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-top: 24px;
+}
+
+.card-price__text {
+  color: var(--text-color-secondary);
+}
+
+.card-price__amount {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.button {
+  margin-top: 16px;
+  height: 40px;
+  padding: 0;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  cursor: pointer;
+  transition: 0.5s;
```

Comment:
It is better to add specific properties to the transition so that the browser does not watch to all properties, e.g.: `transition: color 0.3s ease, background-color 0.3s ease;`

Reply:
I have to learn what is it.